apps/examples/cy4390x_demo/Makefile

```diff
@@ -0,0 +1,120 @@
+############################################################################
+#
+# Copyright 2017, Cypress Semiconductor Corporation or a subsidiary of 
+# Cypress Semiconductor Corporation. All Rights Reserved.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+############################################################################
+
+-include $(TOPDIR)/.config
+-include $(TOPDIR)/Make.defs
+include $(APPDIR)/Make.defs
+
+# cy4390x_demo built-in application info
+
+APPNAME = cy4390x_demo
+PRIORITY = SCHED_PRIORITY_DEFAULT
+STACKSIZE = 20480
```

Comment:
is it required to have such a big stack [ 20KB ] ?

Reply:
this app doesn't use big size memory. but, this app can use big memory because this board contains DDR 128M memory. 

BR, HJ Lee.